src/HonzaBotner.Discord.Services/Commands/BotCommands.cs

```diff
@@ -34,7 +38,7 @@ public BotCommands(ILogger<BotCommands> logger, IOptions<InfoOptions> infoOption
             string status
         )
         {
-            if (type == ActivityType.Custom || type == ActivityType.Streaming)
+            if (type is ActivityType.Custom or ActivityType.Streaming)
```

Comment:
Any reasoning why changing this? I thought you used the later version before and then changed it to the current one. Why change it again? :D

Reply:
That was different file/commit/change where I was checking that something `is not` and it looked too confusing. I can discard this if you want..

Reply:
viz [mega long ultra special link](https://github.com/fit-ctu-discord/honza-botner/pull/181/files#diff-ec6b98a1cc8b72715eaddeca8798d1c6d8e779b8679c6754ae534a2622738bf4R72)